contents/tutorials/cross-domain-tracking.md

```diff
@@ -0,0 +1,95 @@
+---
+title: How to set up cross domain tracking in PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-11-08
+featuredImage: ../images/tutorials/banners/cross-domain-tracking.png
+topics: ['configuration']
+---
+
+Many companies use multiple domains for different parts of their products. A common structure is having one domain for the marketing website and another (subdomain) for the app. Other companies might have multiple connected domains. This is great for having different content and ensuring no conflicts but causes issues when it comes to tracking.
```

Comment:
```suggestion
Many companies use multiple domains for different parts of their products. A common structure is having one domain for the marketing website and another (subdomain) for the app – some also use separate domains documentation and community sections. Other companies might have multiple connected domains. This is great for having different content and ensuring no conflicts but causes issues when it comes to tracking.
```

Optional addition, but I see this quite a bit.

Reply:
Adjusted wording slightly, but added!